tests/pyccel/scripts/print_integrals.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+# ------------------------------- Strings ------------------------------------
+
+from numpy import int32
+if __name__ == '__main__':
+    print(0)
+    print(00)
+    print(1)
+    print(-1)
+    print(-0)
+    print(10000)
+    print(-10000)
+    print(2147483647)
+    print(-2147483648)
+    print(2147483648)
+    print(9223372036854775807)
```

Comment:
Beware printing such large numbers. As they are not in cast statements they will automatically be treated as python ints. The default python ints are 64bit on linux/mac and 32bit on windows.
So a default int cannot be larger than 2147483647 or smaller than -2147483648 if you want windows tests to pass

Reply:
owww okay didn't know that. Good to know thank you

Reply:
> and about the large numbers should I remove those test or just wrap them on int64()? what do you think is best

Wrapping them in int64 sounds like a good idea